.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
```

Comment:
Add a comment notice about whether or not to ask for technical support via bug reports.

Reply:
I guess users could use the same form when submitting bug reports or asking for technical support? Unless we want to have separate sections for these?

Reply:
I recommend being very careful about using GitHub's issues for technical support. We should set up a Discussions thing and point users there, I think.

Reply:
Thanks for the suggestion. I've added a comment to redirect technical support enquiries to Discussions. 
https://github.com/osrf/rmf_demos/pull/184/commits/ba264e4b71a8c32f03ec6770f7c4baba7d149e1b